test/differentials.jl

```diff
@@ -49,6 +49,25 @@
         @test conj(o) == o
     end
 
+	@testset "Thunk" begin
+		@test @thunk(3) isa Thunk
+
+		@testset "show" begin
+			rep = repr(Thunk(rand))
```

Comment:
Indentation seems off in this file

Reply:
oh gross, my vim settigns must have got messed up